apps/condo/pages/employee/index.tsx

```diff
@@ -133,7 +135,7 @@ export const EmployeesPageContent = ({
 const EmployeesPage = () => {
     const { link: { role = {} }, organization }  = useOrganization()
     const userOrganizationId = get(organization, 'id', null)
-    const canManageEmployee = get(role, 'canInviteNewOrganizationEmployees', null)
+    const canManageEmployee = get(role, 'canManageEmployees', null)
```

Comment:
It seems you need to leave the previous option with `canInviteNewOrganizationEmployees`. 
In the interface, it is not the creation of an employee, but the invitation